python/cuml/dask/common/base.py

```diff
@@ -133,7 +133,8 @@ def _check_internal_model(model):
                 raise ValueError("Dask Future expected to contain cuml.Base "
                                  "but found %s instead." % model.type)
 
-        elif model is not None and not isinstance(model, Base):
+        elif (model is not None and
+              not isinstance(model, (Base, experimentalBase))):
```

Comment:
Do we need to update the error message here, considering that it could also be `experimentalBase` ?

Reply:
`experimentalBase` now inherits from `Base`, I will remove any mention of `experimentalBase` except in the test `test_base` that tests the documentation.